Makefile.am

```diff
@@ -49,8 +49,8 @@ pkgconfigdir = @pkgconfigdir@
 errnames_txt_files = 
 
 external_subdirs = @mplsrcdir@ @hwlocsrcdir@ @jsonsrcdir@ @yaksasrcdir@ @pmisrcdir@
-external_ldflags = @mpllibdir@ @yaksalibdir@
-external_libs = @WRAPPER_LIBS@
+external_ldflags = @mpllibdir@ @yaksalibdir@ @MPICHLIB_LDFLAGS@
+external_libs = @WRAPPER_LIBS@ @MPICHLIB_LIBS@
```

Comment:
What if we need `MPICHLIB_LDFLAGS` during configure?

Reply:
I was trying to be too clever. I switched back to a clean revert. Still need to modify the wrapper scripts so these flags won't leak into them.

Reply:
Is the solution just to remove the line adding `LDFLAGS` to the wrapper scripts in the case where interlib dependencies are not supported? I guess an issue could be that users can supply `LDFLAGS=-Lpath/to/lib LIBS=-lfoo` to find dependencies, rather than using `--with-foo=path/to/lib`, but that seems like not a great thing to support.

Reply:
I guess if interlib dependencies are not supported, we have to add the entire LDFLAGS to the wrapper scripts (thus, no change to the current wrappers). There is no way for us to tell which part of the flags used during compilation are needed or not needed when building a user program due to missing interlib dependencies.

Reply:
Alright, then I suppose this is ready for review.